src/components/Item/Item.module.css

```diff
@@ -0,0 +1,74 @@
+
+.itemWrap {
+    width: 90%;
+    min-height: 30px;
+    margin-bottom: 10px;
+    padding: 5px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid grey;
+}
+
+.label {
+    flex: 1;
+    font-size: 14px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/alde_react/pullRequest?prid=5576104)